docker-compose.yml

```diff
@@ -55,6 +55,7 @@ services:
       PS_DEV_MODE: ${PS_DEV_MODE:-1}
       PS_ENABLE_SSL: ${PS_ENABLE_SSL:-0}
       PS_ERASE_DB: ${PS_ERASE_DB:-0}
+      PS_USE_DOCKER_MAILDEV: ${PS_USE_DOCKER_MAILDEV:-1}
```

Comment:
should we enable or disable it by default ?

Reply:
As this is for development I would say enable by default.

Reply:
okey.
So maybe we need some updates for automated tests. (will check)